src/application-navigation/ApplicationNavigationWorkspaceItem.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Could ApplicationNavigationWorkspaceItem just be WorkspaceItem?

Reply:
Removed, and passed through the WorkspaceItem.